### PR TITLE
Refactored stock->TA 

### PR DESCRIPTION
**Pull request by jmaslek**
Wanted to push these changes before #663 to avoid having to go through merging.

No new functionality, just moved argparse into controller.

### CODE REVIEW

File: gamestonk_terminal/stocks/technical_analysis/ta_controller.py [hunk 1]
```diff
@@ -79,53 +79,48 @@ def __init__(
...
-            "/stocks/technical_analysis"
-        )
-        s_intraday = (f"Intraday {self.interval }", "Daily")[self.interval == "1440min"]
-
+        s_intraday = (f"Intraday {self.interval}", "Daily")[self.interval == "1440min"]
         if self.start:
-            print(
-                f"\n{s_intraday} Stock: {self.ticker} (from {self.start.strftime('%Y-%m-%d')})"
-            )
+            str1 = f"\n{s_intraday} Stock: {self.ticker} (from {self.start.strftime('%Y-%m-%d')})"
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/technical_analysis/ta_controller.py:84]: can we have s_stock instead?

File: gamestonk_terminal/stocks/technical_analysis/ta_controller.py [hunk 2]
```diff
@@ -174,19 +169,141 @@ def call_quit(self, _):
...
+            "-i",
+            "--interval",
+            action="store",
+            dest="interval",
+            type=str,
+            default="",
+            choices=["1M", "1W", "1d", "4h", "1h", "15m", "5m", "1m"],
+            help="""Interval, that corresponds to the recommendation given by tradingview based on technical indicators.
+            See https://python-tradingview-ta.readthedocs.io/en/latest/usage.html""",
+        )
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/technical_analysis/ta_controller.py:258]: Can we add export?

File: gamestonk_terminal/stocks/technical_analysis/ta_controller.py [hunk 3]
```diff
@@ -174,19 +169,141 @@ def call_quit(self, _):
...
-        finnhub_view.pattern_recognition_view(other_args, self.ticker)
+
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="pr",
+            description="""
+            Display pattern recognition signals on the data. [Source: https://finnhub.io]
+        """,
+        )
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/technical_analysis/ta_controller.py:284]: Can we add export?

File: gamestonk_terminal/stocks/technical_analysis/ta_controller.py [hunk 4]
```diff
@@ -174,19 +169,141 @@ def call_quit(self, _):
...
+            "-r",
+            "--resolution",
+            action="store",
+            dest="resolution",
+            type=str,
+            default="D",
+            choices=["1", "5", "15", "30", "60", "D", "W", "M"],
+            help="Plot resolution to look for pattern signals",
+        )
+
```
Comment (DidierRLopes) [inline gamestonk_terminal/stocks/technical_analysis/ta_controller.py:295]: Can we add export?

Comment (jmaslek) [reply gamestonk_terminal/stocks/technical_analysis/ta_controller.py:295]: think you commented this twice on this command.  If you meant to comment on the `summary` command then that is a string output, so not exportable based on our conversation in disc.

**Review by DidierRLopes — APPROVED** [review]

**Review by aia — APPROVED** [review]

**Pull request merged**